Nav.php

```diff
@@ -202,6 +202,32 @@ public function renderItem($item)
     }
 
     /**
+     * Check if parent item is visible
+     * @param array $parentItem
+     * @return boolean
+     */
+    protected function isParentVisible($parentItem)
+    {
+        if (isset($parentItem['visible']) && !$parentItem['visible']) {
+            return false;
+        }
+
+        $items = ArrayHelper::getValue($parentItem, 'items');
+        if (empty($items)) {
```

Comment:
In this case `testEmptyItems` is failing because we got `null` and `foreach` try to run on `null`.

We can add additional check for `null` but this is shorter.